src/cdk/v2/destinations/pinterest_tag/procWorkflow.yaml

```diff
@@ -179,10 +179,8 @@ steps:
         eventNames = $.convertToSnakeCase(eventNames);
         eventNames.length === 0 ? eventNames = $.ecomEventMaps.(){eventInLowerCase in .src}.dest[] ?? [];
         (eventNames.length === 0 && .destination.Config.sendAsCustomEvent) ? eventNames = ["custom"];
```

Comment:
Is this toggle `destination.Config.sendAsCustomEvent` still required ? Can't we consider the events as custom ones if they are not part of `ecomEventMaps` ?

Reply:
If this toggle is enabled then we send the event name "custom" which is one of the allowed events in Pinterest. If this toggle is disabled we will send the event name as it is.